frontend/components/feedme/FolderTreeItem.tsx

```
@@ -0,0 +1,260 @@
+
+import React, { useCallback, useState, useRef } from 'react'
+import { ChevronRight, ChevronDown, Folder, FolderOpen, MoreHorizontal, MessageCircle } from 'lucide-react'
+import { useFoldersActions } from '@/lib/stores/folders-store'
+import { useUISidebar, useUIActions } from '@/lib/stores/ui-store'
+import { cn } from '@/lib/utils'
+import { Button } from '@/components/ui/button'
+import { Badge } from '@/components/ui/badge'
+import { FolderContextMenu } from './FolderContextMenu'
+import tinycolor from 'tinycolor2'
+
+interface Folder {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Folder'. Consider to delete it or rename it.

'Folder' is defined here:


(lint/suspicious/noRedeclare)